frontend/package.json

```diff
@@ -31,5 +38,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "proxy": "http://localhost:4000"
```

Comment:
I don't think we've really thought about hosting model too much at this point. But there is a couple of ways to serve the apps.
1. Something serving the ui on the domain and routing stuff in routes under ```/api``` etc.. to the backend
2. API & the UI on separate host names (ex: api.example.com).

It doesn't particularly matter ATM since we haven't decided, but if its option 2 we don't need to proxy requests to the back-end like this. So can remove this line.

If we did go with 2 then this is the wrong port, should be 1338 (https://github.com/emseers/eelbot-ui/blob/23432054fe818495ae8a64ae3934884378099288/backend/app.py#L51)